examples/sdk/train_api.ipynb

```diff
@@ -12,12 +12,13 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        name="huggingface-test",
</pre><p> </p><p>Should we keep using separate name so that we can concurrently deploy jobs with S3 example?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/training-operator/pull/2006/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
created a separate file